sources/packages/backend/apps/queue-consumers/src/services/cas-supplier/cas-evaluation-result-processor/cas-evaluation-result-processor.ts

```diff
@@ -25,4 +28,48 @@ export abstract class CASEvaluationResultProcessor {
     evaluationResult: CASEvaluationResult,
     summary: ProcessSummary,
   ): Promise<ProcessorResult>;
+
+  /**
+   * Process the errors that occurred during processing.
+   * @param studentSupplier student supplier information from SIMS.
+   * @param summary current process log.
+   * @param error errors that occurred during processing.
+   * @param auditUserId audit user id.
+   * @param partialSupplier partial supplier information.ß
+   * @returns processor result.
+   */
+  async processBadRequestErrors(
+    studentSupplier: StudentSupplierToProcess,
+    summary: ProcessSummary,
+    error: string[],
+    auditUserId: number,
+    partialSupplier?: Partial<CASSupplier>,
```

Comment:
A type can be created in the models to be explicit about what the method should receive as shown below.
```ts
export type CASSupplierInfoForBadRequest = Pick<
  CASSupplier,
  "supplierNumber" | "supplierName" | "status" | "supplierProtected"
>;
```